packages/xo-web/src/common/action-button.js

```diff
@@ -182,6 +182,17 @@ export default class ActionButton extends Component {
       </Button>
     )
 
-    return tooltip ? <Tooltip content={tooltip}>{button}</Tooltip> : button
+    return tooltip ? (
+      <Tooltip content={tooltip}>
+        {props.disabled ? (
+          // disabled button should be wrapped before wrapping it with Tooltip
```

Comment:
Why?

Reply:
The tooltip is not displayed in chrome.
[It's already fixed here](https://github.com/vatesfr/xen-orchestra/commit/b13f91ec8dce7d9cb056ce7e33b9f588fa07b994).
https://github.com/vatesfr/xen-orchestra/blob/cfb8d790493aaac9ec7cb693b66bfd5dc1e662ad/packages/xo-web/src/common/tooltip/index.js#L60-L65

Reply:
The commit message contains more info, maybe it should be put in this comment.